sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -1281,7 +1284,7 @@ def get_module_fw_mgmt_feature(self, verbose = False):
         elapsedtime = time.time()-starttime
         logger.info('Get module FW upgrade features time: %.2f s\n' %elapsedtime)
         logger.info(txt)
-        return {'status': True, 'info': txt, 'feature': (startLPLsize, maxblocksize, lplonly_flag, autopaging_flag, writelength)}
+        return {'status': True, 'info': txt, 'result': (startLPLsize, maxblocksize, lplonly_flag, autopaging_flag, writelength)}
```

Comment:
change the field from `feature` to `result` will break the method caller!
e.g. https://github.com/sonic-net/sonic-utilities/blob/84cb00a4b2d7e8fb2bcab259367836fa11a17d0a/sfputil/main.py#L1528

Reply:
@rqx110 thanks for your review and comment, I will change it back to feature.